src/local-development/helpers/fs.ts

```diff
@@ -0,0 +1,19 @@
+import * as vscode from 'vscode';
+import { Uri } from 'vscode';
```

Comment:
Not needed to import Uri separatly.

```suggestion
import * as vscode from 'vscode';
```